common.lua

```diff
@@ -4,5 +4,7 @@
 function convert_to_uf2(target)
     local firmware = target:targetfile()
     os.execv("llvm-strip", { firmware, "-o", firmware .. ".strip" })
-    os.execv("uf2conv", { firmware .. ".strip", "-f0x6CE29E60", "-co", firmware .. ".uf2" })
+    os.execv("uf2conv", { firmware .. ".strip", "-b0x00002000", "-f0x6CE29E60", "-co", firmware .. ".slot1.uf2" })
```

Comment:
Why 0x2000?

Reply:
We discussed this in person: 0x2000 offset was used to mirror the address being used by `uf2conv`, which is what we were using to generate UF2s in our build process. The lower 0x2000 bytes are not being used, so I'll just update this to use a `0x0000` offset so long as that appears to be working.

Reply:
Still seems to work with `0x0000` offsets so after discussion I'm changing to that now.